[21기 박준우] Add: 상품 상세페이지 레이아웃 및 지도 API 불러오기 완료

AutumnWithJay opened the pull request:
## 수정 사항 간략한 한줄 요약
- Add: 상품 상세페이지 레이아웃 및 지도 API 불러오기 완료
		 
## 수정 사항들 자세한 내용
- 상품 상세페이지 레이아웃 완료
- Kakao 지도 API 호출 완료

## 기타 질문 및 특이 사항
- 지도 창 크기에 대한 수정이 이루어질 예정입니다.

## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [x] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [x] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [x] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [x] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.
- [x] Git rebase와 squash를 했고 커밋 수가 하나 인것을 확인 했습니다.

AutumnWithJay committed:
Fix: Resolve Conflict
- Fix: 피드백 사항 수정 완료

dipsiiiiiiiiii commented on src/pages/Detail/Detail.js:
```diff
@@ -1,6 +1,181 @@
+  const location = useHistory();
+  const [clicked, setClicked] = useState(false);
+
+  console.log(location);
```
콘솔 지워 주세요!

AutumnWithJay replied:
수정했습니다😄

dipsiiiiiiiiii commented on src/pages/Detail/Detail.js:
```diff
@@ -1,6 +1,181 @@
+  const [clicked, setClicked] = useState(false);
+
+  console.log(location);
+  const handleClick = () => {
+    clicked ? setClicked(false) : setClicked(true);
+  };
```
clicked 자체가 조건이기 때문에 줄여볼 수 있겠네요!

AutumnWithJay replied:
수정했습니다😄

dipsiiiiiiiiii commented on src/components/Map/Map.js:
```diff
@@ -0,0 +1,39 @@
+
+const Map = props => {
+  useEffect(() => {
+    var container = document.getElementById('map');
+    var options = {
+      center: new kakao.maps.LatLng(37.365264512305174, 127.10676860117488),
+      level: 3,
+    };
+    var map = new kakao.maps.Map(container, options);
+    var geocoder = new kakao.maps.services.Geocoder();
```
공식문서에서는 어떤 환경에서든 사용할 수 있게 하기 위해 ```var```를 사용하지만 저희는 바꿔서 사용해 주세요!

AutumnWithJay replied:
변경했습니다 😄

dipsiiiiiiiiii reviewed (commented):
고생하셨습니다! 리뷰 반영해 주세요!

dipsiiiiiiiiii commented on src/pages/Detail/Detail.js:
```diff
@@ -1,6 +1,180 @@
+  const [clicked, setClicked] = useState(false);
+
+  const handleClick = () => {
+    setClicked(clicked ? 0 : 1);
```
0, 1을 사용하신 이유가 있을까요?

AutumnWithJay replied:
`false, true`로 변경했습니다 😄

dipsiiiiiiiiii reviewed (commented):
리뷰 반영해 주세요!

AutumnWithJay committed:
Fix: Resolve Conflict
- Fix: 피드백 사항 수정 완료
- Fix: 피드백 사항 수정 완료
- Fix: 피드백 사항 수정 완료
- Fix: 북마크 삭제
- Fix: 북마크 삭제
- Fix: 변경 사항 수정
- Fix: Resolve Conflict
- Fix: 피드백 사항 수정 완료
- Fix: 북마크 삭제
- Fix: 북마크 삭제
- Fix: 변경 사항 수정
- Fix: 피드백 사항 변경 완료
- Fix: 피드백 사항 수정 완료
- Fix: 피드백 사항 수정 완료
- Fix: 북마크 삭제
- Fix: 북마크 삭제
- Fix: 변경 사항 수정
- Fix: Resolve Conflict
- Fix: 피드백 사항 수정 완료
- Fix: 북마크 삭제
- Fix: 북마크 삭제
- Fix: 변경 사항 수정
- Fix: 피드백 사항 변경 완료
- Fix: 피드백 사항 수정완료

dipsiiiiiiiiii commented on src/pages/Detail/Detail.js:
```diff
@@ -1,6 +1,180 @@
+            src="/icon/black_bookmark.png"
+            onClick={handleClick}
+          />
+        )}
```
여기도 src의 값만 바뀌기 때문에 줄여볼 수 있을 것 같습니다!
